experiments/compositions/composition/tests/data/TestSimpleCompositionAddFacadeField/input.yaml

```diff
@@ -108,7 +108,6 @@ spec:
   inputAPIGroup: pconfigs.facade.foocorp.com
   expanders:
   - type: jinja2
-    version: v0.0.1.alpha
```

Comment:
so the version is optional for now. If not specified it picks the latest ?

Reply:
Yes, so for out testcases, I removed some version specification and some I left it there.